### ISSUE TITLE
Some config keys missing from the Settings UI

### ISSUE THREAD

**Issue by dreglad** [minio/console#1062]
There are some server config keys missing in the settings UI. In particular, I'd like to have the `api` settings in order to set CORS allowed origins and throttling parameters.

I guess there might be a reason for excluding these in the UI. Maybe to avoid conflicts with config managed by the environment or something like that? If that is the case, having them at least as read-only would be great.

If there is no particular reason not have them in the UI, I can provide with a Pull Request to add them all or some of them.

---

All currently available/missing config keys are:

```
KEYS:
region
storage_class         (missing from UI)
cache
compression
etcd
identity_openid
identity_ldap
identity_tls          (missing from UI)
api                   (missing from UI)
heal                  (missing from UI)
scanner               (missing from UI)
logger_webhook
audit_webhook
audit_kafka
notify_webhook
notify_amqp
notify_kafka
notify_mqtt
notify_nats
notify_nsq
notify_mysql
notify_postgres
notify_elasticsearch
notify_redis
subnet

```

**Comment by dvaldivia** [comment]
These should be there, we will add them in a subsequent PR, we also welcome PRs.

**Comment by dreglad** [comment]
Adding the missing keys in the UI is fairly simple (just update the `portal-ui/screen/Console/Configurations/utils` module), however, the console API is returning incorrectly parsed key values:

For the `api` subsystem, it currently parses:
```
&{api [{requests_max 0} {requests_deadline 0 cluster_deadline=1m} {cors_allow_origin *} {remote_transport_deadline 2h list_quorum=optimal} {replication_workers 250} {replication_failed_workers 8}]}
```

While we should expect this structure:
```
&{api [{requests_max 0} {requests_deadline 0} {cluster_deadline=1m} {cors_allow_origin *} {remote_transport_deadline 2h} {list_quorum=optimal} {replication_workers 250} {replication_failed_workers 8}]}
```

Leading to wrong UI rendering:
![image](https://user-images.githubusercontent.com/50302/134058324-e2a85aa3-ccbc-4402-8828-0c66ab406ea5.png)

After investigating, it turns out that the console API relies on the Help Text structure from the Admin API to parse the available  key values, but there are some missing keys, just the ones that got incorrectly parsed (cluster_deadline and list_quorum)

I made a PR: https://github.com/minio/minio/pull/13255. However they commented out that these keys are not added on purpose.

**Comment by bexsoft** [comment]
Merged in #1064, closing this issue accordingly
